modules/travel_pay/app/services/travel_pay/token_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class TokenService
+    #
+    # GET TOKENS
+    # returns a hash containing the veis_token & btsss_token
+    ##
+    #
+    def get_tokens(current_user)
+      veis_token = token_client.request_veis_token
+      btsss_token = token_client.request_btsss_token(veis_token, current_user)
+
+      { 'veis_token' => veis_token,
+        'btsss_token' => btsss_token }
```

Comment:
Consider:

```ruby
{ veis_token:, btsss_token: }
```

Personally I like using symbols over strings as keys where ever possible.

Reply:
The syntax looks wonky, but similar to javascript's self-referencing keys.

Reply:
I tried to do something like JS keys, but it threw up. 🤣  Looks like I was missing the `:` at the end to make it work.